Code/Tools/AssetProcessor/native/utilities/BuilderManager.cpp

```diff
@@ -29,7 +29,7 @@ namespace AssetProcessor
     //! Amount of time in seconds to wait for a builder to start up and connect
     // sometimes, builders take a long time to start because of things like virus scanners scanning each
     // builder DLL, so we give them a large margin.
-    static const int s_StartupConnectionWaitTimeS = 300;
+    static const int s_StartupConnectionWaitTimeS = 900;
```

Comment:
is this something we can quickly add to a setreg file for an over ride?

Reply:
Added